src/Analyzers/Core/Analyzers/UseCollectionInitializer/UpdateExpressionState.cs

```diff
@@ -123,16 +123,15 @@ public bool ValuePatternMatches(TExpressionSyntax expression)
     /// <summary>
     /// Analyze an expression statement to see if it is a legal call of the form <c>val.Add(...)</c>.
     /// </summary>
-    public bool TryAnalyzeAddInvocation<TExpressionStatementSyntax>(
-        TExpressionStatementSyntax statement,
+    public bool TryAnalyzeAddInvocation(
```

Comment:
changed this to actually only analyze the invocation-expression itself, not hte containing expression-statement.  that's because in the next PR `.Add/.AddRange` may be part of a long expression chain, not standalone like `foo.Add(x);`